crates/genesis/src/lib.rs

```diff
@@ -217,6 +224,55 @@ pub struct GenesisAccount {
         deserialize_with = "deserialize_storage_map"
     )]
     pub storage: Option<HashMap<B256, B256>>,
+    /// The account's private key.
+    #[serde(
+        rename = "secretKey",
+        default,
+        skip_serializing_if = "Option::is_none",
+        with = "secret_key"
+    )]
+    pub private_key: Option<LocalWallet>,
```

Comment:
Can we just do B256?

Reply:
yeah would prefer that actually, much easier to simplify. wdyt @mattsse

Reply:
@Evalir I'd actually prefer just B256 as well.

point in my review still stands, we should have functions on LocalWallet that take care of that